install/tools/man/ipa-migrate.1

```diff
@@ -0,0 +1,112 @@
+.\"
+.\" Copyright (C) 2023  FreeIPA Contributors see COPYING for license
+.\"
+.TH "ipa-migrate" "1" "Fed 1 2024" "IPA" "IPA Manual Pages"
+.SH "NAME"
+ipa\-migrate \- Migrate an IPA server from one machine to another
+.SH "SYNOPSIS"
+ipa\-migrate
+.SH "DESCRIPTION"
+
+Use the \fIipa-migrate\fR command to migrate one
+IPA server to an existing local IPA server installation.
+
+Migrate IPA schema, configuration, and database to a local IPA server.  This
+migration can be done online, where the tool will query the remote server. Or,
+offline where LDIF files can be provided.  You can mix and match online and
+offline. So for example you could migrate the schema and configuration online,
+and then use an exported LDIF file for the database migration portion (this
+might be more useful for very large databases as you don't need to worry about
+network interruptions)
+
+
+.SH POSITIONAL ARGUMENTS
+.TP
+\fBprod\-mode\fR
+In this mode everything will be migrated including the current user SIDs and
+DNA ranges
+.TP
+\fBstage\-mod\fR
+In this mode, SIDs & DNA ranges are not migrated, and DNA attributes are reset
+.SH "COMMANDS"
+.TP
+\fB\-v\fR, \fB\-\-verbose\fR
+Use verbose output while running the migration tool.
+.TP
+\fB\-e\fR, \fB\-\-hostname=HOSTNAME\fR
+The host name of the remote IPA server that is being migrated from.
+.TP
+\fB\-D\fR, \fB\-\-bind\-dn=BIND_DN\fR
+The Bind DN (Distinguished Name) or an LDAP entry to bind to the remote IPA server with.
+Typically this is "cn=directory manager", but it could be any entry that has
+access to read the userPassword attribute.
+.TP
+\fB\-w\fR, \fB\-\-bind\-pw=PASSWORD\fR
+The password for the Bind DN that is authenticating against the remote IPA server.  If
+a password is not provided then the tool with prompt for the password if needed.
```

Comment:
I think it is worth mentioning that the bind user needs read access to the userPassword attribute if they want to migrate passwords. This is why DM is the default. I think adding the default here would be useful as well.